definitions/consented/json/CaseEvent.json

```diff
@@ -367,7 +367,7 @@
     "Description": "Create General Order",
     "DisplayOrder": 14,
     "PreConditionState(s)": "*",
-    "PostConditionState": "*",
+    "PostConditionState": "orderMade",
```

Comment:
What events can be triggered from orderMade or orderDrawn?
Do these need to be added to precondition state for the close event?

Reply:
shouldn't need added to close as close event preconditions are
    "PreConditionState(s)": "*",
for contested and order made is in the consent order preconditions

from order drawn a court admin can..
![image](https://user-images.githubusercontent.com/12084873/87942263-3fa57500-ca94-11ea-9233-8cea11db33f3.png)